crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space/bootstrap_task.rs

```diff
@@ -84,7 +84,8 @@ impl BootstrapTask {
     ) -> Arc<RwLock<Self>> {
         let task_this = this.clone();
         tokio::spawn(async move {
-            let backoff_multiplier = if bootstrap_check_delay_backoff_multiplier < 2 {
+            let backoff_multiplier = if !cfg!(test) && bootstrap_check_delay_backoff_multiplier < 2
```

Comment:
`cfg!(test)` is a bit broad here. It's breaking a test in the bootstrap test suite.

Is this still needed with bootstrap disabled? If so, is there a way to solve this with config instead?